book/src/proposals/simple-payment-and-state-verification.md

```diff
@@ -58,17 +58,17 @@ Clients that want to query the chain for a receipt of the "latest" state would n
 
 Leaders should coalesce the validator votes by stake weight into a single entry. This will reduce the number of entries necessary to create a receipt.
 
-### Chain of Entries
+### Chain of Components in Merkle Path
 
-A receipt has a PoH link from the payment or state Merkle Path root to a list of consecutive validation votes.
+A receipt has a PoH link from the Payment or State Merkle Path Root to a list of consecutive validation votes.
```

Comment:
Consistent capitalization.

Reply:
I'm confused on what capitalization means here.  I'd think "Path" could be "path" and that only Merkle (a proper noun) needs to be capitalized.  If those things you're capitalizing are data structures in the codebase, they capitalizing them makes sense, but should also be surrounded with backticks.